src/snarky.node.js

```diff
@@ -10,18 +10,21 @@ export {
   isReady,
 };
 let { proxyClasses } = require('./proxyClasses.js');
-let snarkyServer = require('./node_bindings/snarky_js_node.bc.js');
+let snarky = require('./node_bindings/snarky_js_node.bc.js');
 let snarkySpec = require('./snarky-class-spec.json');
 
+let { picklesCompile } = snarky;
```

Comment:
I think it would be really helpful to have a short comment above on what `picklesCompile` is doing. Since it's defined in the OCaml side of things, it would be nice to know what it does without swapping to a different project.

Reply:
addressed here: https://github.com/o1-labs/snarkyjs/pull/123/commits/f0014217b9911438da621751a33f7bd3c97f3586